README.md

```diff
@@ -45,6 +45,29 @@ $ npm run dev
 
 Now visit the page `http://localhost:8080/`
 
+# How to run deploy through bothub
+
+First install bothub-cli
+```
+$ pip install bothub-cli
+```
+Create the .bothub folder in C:\Users\User\
+Be sure to get the needed credentials(config.yml) and put it in the .bothub folder
+```
+mkdir ~/.bothub
+```
+Navigate to the MyBot directory
```

Comment:
"Open the project folder and navigate to the MyBot directory" might be a better way to word it so that we're not looking for MyBot in the .bothub file.